stub_uploader/build_wheel.py

```diff
@@ -404,21 +416,29 @@ def main(
     created after uploading it.
     """
     ts_data = read_typeshed_data(Path(typeshed_dir))
+    # TODO: Merge BuildData and PackageData and encapsulate MetaData.
+    # See also https://github.com/typeshed-internal/stub_uploader/issues/123.
```

Comment:
The doc messes up with some url interpreters

```suggestion
    # See also https://github.com/typeshed-internal/stub_uploader/issues/123 .
```
or
```suggestion
    # See also https://github.com/typeshed-internal/stub_uploader/issues/123
```

---

For example:
![image](https://github.com/typeshed-internal/stub_uploader/assets/1350584/75e8c354-985e-4746-86f1-bc172fccd825)